css-canvas-medical/styles.css

```diff
@@ -0,0 +1,267 @@
+/* Header */
+body {
+  width: 1380px;
+  margin: 0 auto;
+}
```

Comment:
Rather than constrain the width of the body, try creating a `.page-container` class.